package.json

```diff
@@ -10,7 +10,8 @@
     "@babel/plugin-syntax-import-meta": "^7.0.0",
     "@babel/plugin-transform-flow-strip-types": "^7.0.0",
     "@babel/preset-env": "^7.1.5",
-    "babel-plugin-transform-invariant-location": "^1.0.0"
+    "babel-plugin-transform-invariant-location": "^1.0.0",
+    "caniuse-lite": "^1.0.30000912"
```

Comment:
Looking at the changes in `package-lock.json` it seems we did have a `caniuse-lite` package but here it seems to be saying that we are adding it for the first time 🤔

Reply:
`@babel/preset-env` depends on `browserlist`, which already depends on `caniuse-lite`

Reply:
Yes this is a case of dependency pinning. While `browserlist` depends on `caniuse-lite` it depends on a lower version. We want to ensure the lower version doesn't get installed, and instead we install a minimum version of `1.0.3000912`